pkg/scaffold/v2/controller/controller_suitetest.go

```diff
@@ -32,16 +33,30 @@ var _ input.File = &ControllerSuiteTest{}
 type ControllerSuiteTest struct {
 	input.Input
 
-	// Resource is the resource to scaffold the controller_kind_test.go file for
+	// Resource is the Resource to make the Controller for
 	Resource *resource.Resource
+
+	// ResourcePackage is the package of the Resource
+	ResourcePackage string
```

Comment:
This was removed in #1219. See below.

Reply:
Please, see: https://github.com/kubernetes-sigs/kubebuilder/pull/1219/files#r363710055

Reply:
Closing the same comment/obs was made 3 times. Just one to address your POV I think is enough :-)